src/types/ptr_type.rs

```diff
@@ -9,7 +9,6 @@ use crate::values::{ArrayValue, AsValueRef, IntValue, PointerValue};
 use crate::AddressSpace;
 
 use crate::types::enums::BasicMetadataTypeEnum;
-use std::convert::TryFrom;
 use std::fmt::{self, Display};
 
 /// A `PointerType` is the type of a pointer constant or variable.
```

Comment:
Should get_element_type be removed from PtrType in 15? Sees like a breach of safety if opaque ptrs are enabled?

Reply:
yes it should

Reply:
Fixed!